apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -672,6 +674,87 @@ export const transactionWorkflow = workflow(
   }
 );
 
+export const myBucket = bucket("myBucket");
+export const bucketSignal = signal<{ data: string }>("bucketSignal");
+
+export const myBucketStream = myBucket.stream(
```

Comment:
is `stream` the right word here?

Reply:
Nope, but I picked a word until we could debate it.

Notify? (They call them notifications)
OnNotification
OnChange
Changes
OnPut/OnDelete

Reply:
Could do idiomatic node js APIs. 

```
bucket.on("put")
```

Reply:
We could, though I'd like to be able to handle multiple event types from one handler, in an arbitrary set.

Reply:
Can always pass an array .on(["put"])

Reply:
Yeah, thats what I was thinking too.